### ISSUE TITLE
Are the `vc.type` elements in the examples being generated properly?

### ISSUE THREAD

**Issue by jdkizer9** [HL7/fhir-shc-vaccination-ig#206]
Scenario 1 bundle (COVID-19 immunization) [expanded JWS](https://github.com/HL7/fhir-shc-vaccination-ig/blob/eda4d8319cd747a127e10f9749c76199241642de/examples/Scenario1Bundle-jws-payload-expanded.json#L5) includes the following type:

```
"type": [
      "https://smarthealth.cards#health-card",
      "https://smarthealth.cards#labresult"
    ],
```

Scenario 3 bundle (COVID-19 lab result) [expanded JWS](https://github.com/HL7/fhir-shc-vaccination-ig/blob/eda4d8319cd747a127e10f9749c76199241642de/examples/Scenario3Bundle-jws-payload-expanded.json#L5) includes the same `type` element.

**Comment by nkarip** [comment]
Yes, main branch is not correct. We have a fix pending in the dev branch at https://github.com/HL7/fhir-shc-vaccination-ig/tree/dev/examples but this has not yet promoted to main.

**Comment by p9g** [comment]
See https://github.com/HL7/fhir-shc-vaccination-ig/pull/174 and https://github.com/HL7/fhir-shc-vaccination-ig/issues/172
